helpers/setup.sh

```diff
@@ -61,32 +141,46 @@ function describe_all_resources () {
 
     if [ "${answer}" = "YES" ];
```

Comment:
```suggestion
    if [ "${answer^^}" = "YES" ];
```

That'll convert the value of answer to all uppercase :)